packages/webauthn-pcd/src/WebAuthnPCD.ts

```diff
@@ -0,0 +1,124 @@
+import { PCD, PCDPackage, SerializedPCD } from "@pcd/pcd-types";
+import { startAuthentication } from "@simplewebauthn/browser";
+import {
+  generateAuthenticationOptions,
+  verifyAuthenticationResponse,
+} from "@simplewebauthn/server";
+import {
+  AuthenticationResponseJSON,
+  AuthenticatorDevice,
+} from "@simplewebauthn/typescript-types";
+import JSONBig from "json-bigint";
+import { v4 as uuid } from "uuid";
+
+export const WebAuthnPCDTypeName = "webauthn-pcd";
+
+export interface WebAuthnPCDArgs {
+  rpID: string;
+  origin: string;
+  challenge: string;
+  authenticator: AuthenticatorDevice;
+}
+
+export interface WebAuthnPCDClaim {
+  /**
+   * The challenge that is claimed to be signed by the credential.
+   */
+  challenge: string | ((challenge: string) => boolean);
```

Comment:
having this be a function is not ideal because a PCD should be easily serializable/deserializable

Reply:
makes sense -- i also converted the Uint8Array to a base64 string as that made serialization difficult